homeassistant/components/daikin_madoka/translations/es.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Translations are handled via Lokalise. Additional languages should not be provided. Please remove this file.

Reply:
This used to be a custom component but I decided to work a bit more and make it available to the core. Thank you for your help!

Reply:
@frenck Let me ask you a question... I would like to notify the user that the integration might need to be reloaded if the bluetooth adapter fails. Until now, in my custom component I used a logging message. What would be the correct approach here? Is it enough to use the logs?

Apart from that, do logging messages have to be lokalised? I am not sure as all the logs seem written in English even if my instance is configured in Spanish.